### ISSUE TITLE
Ruby tests fails with latests TS version [JIRA: CLIENTS-1052]

### ISSUE THREAD

**Issue by korry8911** [basho/riak-ruby-client#296]
Master
2.2.5
Riak TS EE 1.5.0rc1
Ubuntu 16.04
bundle exec rspec --tag time_series
 expected no Exception, got #<Riak::ProtobuffsErrorResponse: Expected success from Riak but received 1001. The upper and lower boundaries are equal or adjacent. No results are possible.>

Was running the time_series tests n the new 1.5.0rc1 and got this... I think the error message is due to new changes in 1.5.0. Probably just need to change the expected response from riak in the tests.

Full output:

```
Failures:

  1) Time Series query interface returns an empty collection when not finding data
     Failure/Error: expect{ subject_without_data.issue! }.to_not raise_error

       expected no Exception, got #<Riak::ProtobuffsErrorResponse: Expected success from Riak but received 1001. The upper and lower boundaries are equal or adjacent. No results are possible.> with backtrace:
         # ./lib/riak/client/beefcake/protocol.rb:106:in `expect'
         # ./lib/riak/client/beefcake/time_series_query_operator.rb:23:in `block in query'
         # ./lib/riak/client/beefcake_protobuffs_backend.rb:43:in `protocol'
         # ./lib/riak/client/beefcake/time_series_query_operator.rb:21:in `query'
         # ./lib/riak/time_series/query.rb:40:in `block in issue!'
         # ./lib/riak/client.rb:377:in `block in recover_from'
         # ./vendor/ruby/2.2.0/gems/innertube-1.1.0/lib/innertube.rb:138:in `take'
         # ./lib/riak/client.rb:375:in `recover_from'
         # ./lib/riak/client.rb:347:in `protobuffs'
         # ./lib/riak/time_series/query.rb:38:in `issue!'
         # ./spec/integration/riak/time_series_spec.rb:122:in `block (4 levels) in <top (required)>'
         # ./spec/integration/riak/time_series_spec.rb:122:in `block (3 levels) in <top (required)>'
     # ./spec/integration/riak/time_series_spec.rb:122:in `block (3 levels) in <top (required)>'

Finished in 1.15 seconds (files took 0.51364 seconds to load)
12 examples, 1 failure

Failed examples:

rspec ./spec/integration/riak/time_series_spec.rb:121 # Time Series query interface returns an empty collection when not finding data
```

**Comment by lukebakken** [comment]
Fixed in #297

**Comment by Basho-JIRA** [comment]
Fixed, or closed via GitHub issues. 

***[posted via JIRA by Alexander Moore]***
